### ISSUE TITLE
Problem with adding css class to popup

### ISSUE THREAD

**Issue by kudinov-k** [lifaon74/ngx-popups#2]
Hi, if add css class throw 
`popup.element.classList.add('my-component-popup');`
this class add only after transition end. And this looks not good if I want set size for popup, it's appear with default size, and after trantision finish size change with added class. In this case use css class with configs like it was in old version, I think it's best solution.

In readme and in examples not described how use popups with new ngx-dom-component (input/output decorator)

**Comment by lifaon74** [comment]
Hum I'll need more informations. The update of ngx-dom-component was just for AOT, nothing more. Moreover, check that you have the last version of ngx-popups.

The `popup` component (instance of the class) is available after `open` resolved or into the ngOnChanges of the component itself.

In my opinion, the best practice would be: a popup component should be isolated => import its own style, add by itself its classes, etc... you should avoid setting class or whatever operations after the `open` (in the `then`). This way, you just provide a config to `open` and let the component manage itself.

**Comment by kudinov-k** [comment]
> In my opinion, the best practice would be: a popup component should be isolated => import its own style, add by itself its classes, etc...

Yes I agree, but by default popup has width 500px, and by example my component has width 300px. Popup start opening with width 500px, and after my component loded in popup it's change to 300px. It's looks not good, like glitching. To avoid this I must before opening provide to popup my needed width.

**Comment by lifaon74** [comment]
It's because you're applying the class after the transition end.
You have 2 solutions : 
- use open with waitTransitionEnd set to false => `open(config, false)`
- apply style in ngOnInit or ngOnChanges or your popup component. => prefer this one
